python/ray/tests/test_memory_pressure.py

```diff
@@ -146,12 +138,34 @@ def test_memory_pressure_kill_actor(ray_with_memory_monitor):
     )
 
 
+@pytest.mark.skipif(
+    sys.platform != "linux" and sys.platform != "linux2",
+    reason="memory monitor only on linux currently",
+)
+def test_memory_pressure_kill_restartable_actor(ray_with_memory_monitor):
+    leaker = Leaker.options(max_restarts=1, max_task_retries=1).remote()
+
+    bytes_to_alloc = get_additional_bytes_to_reach_memory_usage_pct(
+        memory_usage_threshold_fraction + 0.1
+    )
+    with pytest.raises(ray.exceptions.RayActorError) as _:
```

Comment:
Isn't it supposed to be `OutOfMemoryError `?

Reply:
the last PR was improving workercrash, we haven't touched actor

Reply:
Yes, I think ideally we should make sure actor OOM errors are consistent with task OOMs, same info, etc. But I think it's fine to leave this for a follow-up since it's not related to this PR.